test/subscribe.test.js

```diff
@@ -0,0 +1,89 @@
+describe('Subscribe', function() {
+  describe('PersistedModel', function() {
+    describe('configured to source changes locally', function() {
+      before(function() {
+        var test = this;
+        var app = loopback({localRegistry: true});
+        var ds = app.dataSource('ds', {connector: 'memory'});
+        this.Score = app.model('Score', {
+          dataSource: 'ds',
+          changeDataSource: false // use only local observers
```

Comment:
That is placeholder for allowing you to specify a remote broker (eg. redis or mosquitto) that allows you to subscribe to changes originating from other servers.

We also might use it to subscribe to changes from a db op log.